src/examples/basic.py

```diff
@@ -9,15 +7,8 @@ class MyEnvClass(NetworkBase):
     """
 
     def create_hook(self):
-        # Add a bastion host as a child stack in the environment
-        self.add_child_template(Bastion())
-
-        # Add a preconfigured ASG + ELB as a child stack in the environment
-        self.add_child_template(HaCluster(
-            name="MyCluster",
-            min_size=2, max_size=3,
-            instance_type='t2.micro'))
-
+        # Do custom troposphere resource creation here
+        super(MyEnvClass, self).create_hook()
```

Comment:
TBH the client I am working w/ had already removed that code.  You should look at the history of this file too, it went from barebone to this.  Probably it's worth it to keep both examples: barebone +  bastion&ha-nat